adoc/Release-Notes.adoc

```diff
@@ -34,14 +34,23 @@ endif::[]
 == Major Changes
 
 [id='sec.1_5_2']
-=== Release 1.5.2, January 2020
+=== Release 1.5.2, February 2020
 
 [id='sec.1_5_2.new']
 ==== What Is New?
 * {scf} has been updated to version 2.20.1:
 ** cf-deployment has been updated to 12.17
 ** Optimized startup time for various roles
 ** Added/fixed podAntiAffinity rules for various roles
+* The Stratos UI has been updated to version 2.7.0:
+** Tech preview for helm feature
```

Comment:
```suggestion
** Tech preview for Helm feature
```

Reply:
this line will be junked once I review the actual 2.7.0 changelog so mark this as incomplete for now ;)